packages/dds/tree/src/shared-tree/treeCheckout.ts

```diff
@@ -756,87 +638,51 @@ export class TreeCheckout implements ITreeCheckoutFork {
 		return this.forest.anchors.locate(anchor);
 	}
 
-	// #region Transactions
-
-	public isTransacting(): boolean {
-		return this.transactions.size !== 0;
-	}
-
-	public startTransaction(): void {
-		this.checkNotDisposed();
-
+	/**
+	 * The {@link Transactor} for this checkout.
```

Comment:
These docs aren't new, I just moved them from where they used to be (on `ITransaction`) so that they don't get lost. I don't think it's appropriate to put them on the `Transactor` interface since that interface lives in core now, and doesn't know about checkouts.